### PR TITLE
Update turn-pronouns-on-or-off.md

### PR DESCRIPTION
**Pull request by PollyNincevic**
Added info about turning the pronouns feature on or off for your organization using MS Graph (with link) - line 62

### CODE REVIEW

File: microsoft-365/admin/add-users/turn-pronouns-on-or-off.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ Use the following steps to turn the pronouns feature on or off in the Microsoft
 3. To turn on the pronouns feature, in the **Pronouns** pane, select the **Turn on and allow pronouns** checkbox. To turn off the pronouns feature, clear the checkbox.
 4. Select **Save**.
 
+To turn the pronouns feature on or off for your organization using MS Graph, see [Manage pronouns settings for an organization using the Microsoft graph API](/graph/pronouns-configure-pronouns-availability.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/turn-pronouns-on-or-off.md]:
```suggestion
To turn the pronouns feature on or off for your organization using Microsoft Graph, see [Manage pronouns settings for an organization using the Microsoft Graph API](/graph/pronouns-configure-pronouns-availability.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PollyNincevic, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@PollyNincevic Please implement the suggested changes. Thanks

**Review by PollyNincevic — COMMENTED** [review]
Updated per suggestions

**Comment by PollyNincevic** [comment]
@yogkumgit can u advise if anything else is needed? Thank u for ur time.

**Pull request merged**